src/png.imageio/png_pvt.h

```diff
@@ -684,6 +684,13 @@ write_info(png_structp& sp, png_infop& ip, int& color_type, ImageSpec& spec,
                      (png_uint_32)(yres * scale), unittype);
     }
 
+#ifdef PNG_eXIf_SUPPORTED
```

Comment:
What determines if PNG_eXIf_SUPPORTED is found? Is it always present in sufficiently new libpng? (If so, how new? Is it always present based on a minimum version that we should try enforcing?)

Reply:
Indeed libpng >= 1.6.32 added support for the eXIf chunk in PNG and defines PNG_EXIf_SUPPORTED. To be honest, I'm not entirely sure if there is a way to build libpng >= 1.6.32 without this support. I took the same approach as is used elsewhere in this file for reading eXIf data.

Reply:
Makes sense.